resources/views/pages/user/[user]/posts.blade.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+use App\Community\Services\ForumRecentPostsPageService;
+use App\Models\User;
+use App\Support\Shortcode\Shortcode;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{name, middleware, render};
+
+name('user.posts');
```

Comment:
I wonder if this should also be `recent-posts` for consistency. In this case, a user's "recent" posts may not be all that recent.

Reply:
I had the same thought, and that ambiguity is what led me to lean towards "user.posts" instead of "user.recent-posts".

I could go either way - do you think it's worth changing? If so, I don't mind.

Reply:
I'll keep this as `user.posts` for the time being, for the reasons you've stated - a user's "recent" posts could have happened months or even years ago. Recency for this specific view is just the sort order (which happens to be hardcoded), but it's not the primary viewing context.

Not a strong opinion though, we can always change it later if it makes more sense to be `user.recent-posts`.